pennylane/noise/conditionals.py

```diff
@@ -267,22 +268,26 @@ def _get_ops(val):
         classes corresponding to val.
     """
     vals = val if isinstance(val, (list, tuple, set)) else [val]
-    return tuple(
-        (
-            getattr(qml.ops, val, None) or getattr(qml, val)
-            if isinstance(val, str)
-            else (val if isclass(val) else getattr(val, "__class__"))
-        )
-        for val in vals
-    )
+    op_names = []
+    for _val in vals:
+        if isinstance(_val, str):
+            op_names.append(getattr(qml.ops, _val, None) or getattr(qml, _val))
+        elif isclass(_val):
+            op_names.append(_val)
+        elif isinstance(_val, (MeasurementValue, MidMeasureMP)):
+            mid_measure = _val if isinstance(_val, MidMeasureMP) else _val.measurements[0]
+            op_names.append(["MidMeasure", "Reset"][getattr(mid_measure, "reset", 0)])
+        else:
+            op_names.append(getattr(_val, "__class__"))
+    return tuple(op_names)
 
 
 def _check_arithmetic_ops(op1, op2):
     """Helper method for comparing two arithmetic operators based on type check of the bases"""
     # pylint: disable = unnecessary-lambda-assignment
 
-    if isinstance(op1, (Controlled, ControlledSequence)) or isinstance(
-        op2, (Controlled, ControlledSequence)
+    if isinstance(op1, (Adjoint, Controlled, ControlledSequence)) or isinstance(
+        op2, (Adjoint, Controlled, ControlledSequence)
```

Comment:
Curious why these were not initially there?

Reply:
This case was kinda being handled by the transform, so was missed here.